lib/attributor/types/class.rb

```diff
@@ -0,0 +1,32 @@
+require_relative '../exceptions'
+
+module Attributor
+  class Class
+    include Type
+
+    def self.native_type
+      return ::Class
+    end
+
+    def self.load(value, context=Attributor::DEFAULT_ROOT_CONTEXT, **options)
+      unless value.kind_of?(::String) || value.nil?
+        raise IncompatibleTypeError,  context: context, value_type: value.class, type: self
+      end
+
+      value = "::" + value if value[0..1] != '::'
+
+      value && eval(value)
```

Comment:
...for security reasons, and convenience actually, I think that being able to say what class it needs to be converted might actually be better.
Maybe described as "Class.of(MyClass)" ...
that would get rid of the constantize for the common case...and perhaps we can only default to that if we support the bare "Class" definition (without the "of")